crap-advisor-frontend/src/app/constants/index.ts

```diff
@@ -0,0 +1,3 @@
+export const apiUrl = "http://25.38.182.18:8082";
+
+export type $needToFix = any;
```

Comment:
Whataheck is this thing?

Reply:
That's my Type Alias to avoid any type while i change js code to ts

Reply:
Great practice